docs/contributing/first-time.md

```diff
@@ -87,6 +87,18 @@ Learn from their mistakes, and don't commit them yourself.
     It also sends notifications to hundreds of developers, informing them that you have not read this guide, and annoying many of them.
     You should instead learn how to {ref}`work-with-github-issues-label` and {ref}`run tests and code quality checks locally <test-and-code-quality-label>`.
 
+4.  **Don't ask if an issue is open.**
+    Instead you can determine whether an issue is open by doing your own research using the following tips.
+
+    -   Check the issue's status indicator for a green label of {guilabel}`Open`.
+    -   Look for linked open pull requests in the issue's history.
+    -   Search open pull requests for the issue.
+        Sometimes contributors fail to link their pull request to an open issue.
+    -   Review unreleased change log entries in the package's {guilabel}`news` directory.
+        Each pull request must have a change log entry, and these entries end up here when pull requests are merged and closed.
+    -   Search release notes to see whether the issue has been resolved.
+        On rare occasions, contributors forget to close the original issue.
```

Comment:
@stevepiercy This list is great, but also misses the obvious one: start volto and follow the steps to reproduce the issue, to see if it still happens.

Reply:
Thanks for the tip. I made it more generic for all of Plone in 9fc70e93186b66bdda0e2e427f8732d7cb206399.